sklearn/tests/test_metaestimators.py

```diff
@@ -36,6 +37,9 @@ def __init__(self, name, construct, skip_methods=(),
                   skip_methods=['transform', 'inverse_transform', 'score']),
     DelegatorData('RFECV', RFECV,
                   skip_methods=['transform', 'inverse_transform', 'score']),
+    DelegatorData('BaggingClassifier', lambda est: BaggingClassifier(est),
+                  skip_methods=['transform', 'inverse_transform', 'score',
+                                'predict_proba', 'predict_log_proba', 'predict'])
```

Comment:
I gather the delegation relationship with `predict_proba` and `predict` is a bit more complicated than the generic test will admit?